iib/workers/tasks/build_merge_index_image.py

```diff
@@ -194,13 +212,24 @@ def handle_merge_request(
     _update_index_image_build_state(request_id, prebuild_info)
     source_from_index_resolved = prebuild_info['source_from_index_resolved']
     target_index_resolved = prebuild_info['target_index_resolved']
+    dockerfile_name = 'index.Dockerfile'
 
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
-        if is_image_fbc(source_from_index_resolved):
-            err_msg = 'File-Based catalog image type is not supported yet.'
+        source_fbc = is_image_fbc(source_from_index_resolved)
+        target_fbc = is_image_fbc(target_index_resolved)
+
+        if source_fbc and not target_fbc:
+            err_msg = (
+                'Cannot merge source File-Based Catalog index image into target SQLite index image.'
+            )
             log.error(err_msg)
             raise IIBError(err_msg)
 
+        if source_fbc:
```

Comment:
is this needed?

Reply:
yes - we want to have this information in logfiles.

Reply:
Minor: These 2 logs are useful, how about we move it before raising  above error "Cannot merge source File-Based Catalog index image into target SQLite index image". It will ease reading logs more

Reply:
I think in that error message you have this info as well:
'Cannot merge **source File-Based Catalog** index image into **target SQLite** index image.'

But I can move it up if you think it is better.

Reply:
Can we add comment that was requested